ros2bag/ros2bag/verb/play.py

```diff
@@ -90,6 +90,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '-p', '--start-paused', action='store_true', default=False,
             help='Start the playback player in a paused state.')
+        parser.add_argument(
+            '--start-time', type=check_positive_float, default=0.0,
```

Comment:
Naming parameter as `--start-time` and mapping it internally to the `start_offset` causing a lot of confusion what it really is.
It could be not clear that `start-time` doesn't really means start time in rosbag timestamp but rather time offset from the begging of the bag.
IMO it would be better to rename parameter `--start-time` to the `--start-offset`.

Reply:
Yes, I agree that `--start-offset` would be less confusing for developers. The reason I initially picked `--start-time` for the CLI argument was because I thought the wording would align with what (most) users expected - time offset from the beginning of the bag.

Having said that, it's better to not leave room for ambiguity. Addressed in https://github.com/ros2/rosbag2/pull/931/commits/975c65c9411d028cfaf7d140423c00e9679306a8.